src/Components/Blueprints/ImportBlueprintModal.test.tsx

```diff
@@ -207,4 +260,79 @@ describe('Import modal', () => {
       ).toBeInTheDocument()
     );
   });
+
+  const getSourceDropdown = async () => {
+    const sourceDropdown = await screen.findByRole('textbox', {
+      name: /select source/i,
+    });
+    await waitFor(() => expect(sourceDropdown).toBeEnabled());
+
+    return sourceDropdown;
+  };
+
+  test('should enable button on toml blueprint and go to wizard', async () => {
+    await setUp();
+    await uploadFile(`blueprints.toml`, ONPREM_BLUEPRINT_TOML);
+    const reviewButton = screen.getByTestId('import-blueprint-finish');
+    await waitFor(() => expect(reviewButton).not.toHaveClass('pf-m-disabled'));
+    user.click(reviewButton);
+
+    await waitFor(async () =>
+      expect(
+        await screen.findByText('Image output', { selector: 'h1' })
+      ).toBeInTheDocument()
+    );
+
+    // Image output
+    await waitFor(
+      async () => await user.click(await screen.findByTestId('upload-aws'))
+    );
+    await clickNext();
+
+    // Target environment aws
+    const radioButton = await screen.findByRole('radio', {
+      name: /use an account configured from sources\./i,
+    });
+    await waitFor(() => user.click(radioButton));
+    const awsSourceDropdown = await getSourceDropdown();
+    await waitFor(() => expect(awsSourceDropdown).toBeEnabled());
+    await waitFor(() => user.click(awsSourceDropdown));
+    const awsSource = await screen.findByRole('option', {
+      name: /my_source/i,
+    });
+    await waitFor(() => user.click(awsSource));
+
+    await clickNext();
+
+    // Registration
+    await screen.findByText(
+      'Automatically register and enable advanced capabilities'
+    );
+    const registrationCheckbox = await screen.findByTestId(
+      'automatically-register-checkbox'
+    );
+    expect(registrationCheckbox).toHaveFocus();
+    await screen.findByRole('textbox', {
+      name: 'Select activation key',
+    });
+    await clickNext();
+
+    // OpenScap
+    await clickNext();
+
+    //File system configuration
+    await clickNext();
```

Comment:
Yeah, we might. :thinking: And I think it could show to us more, what values we are able to import from on prem. :) But would you be okay if we just did that in a followup maybe? :thinking: